package.json

```diff
@@ -56,7 +56,7 @@
     "typescript": "^4.0.2"
   },
   "peerDependencies": {
-    "nock": "^11.0.0"
+    "nock": "^13.0.4"
```

Comment:
I might be misunderstanding here, but why would we want to bump the peerDependency? Wouldn't `^11.0.0` allow people to use v13 if they want?

Reply:
I have changed this to a range, to be explicit about supporting latest version range but keep the support for 11 and more. I hope it's ok. 😄